Python/chapter01/1.1 - Is Unique/jesus_1.1.sol.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+
+def is_unique(s: str) -> bool:
```

Comment:
Can you add a [docstring](https://www.python.org/dev/peps/pep-0257/)? In the docstring, please document what the runtime+space complexity is.